Rakefile

```diff
@@ -20,19 +20,6 @@ Rake::TestTask.new(:testing) do |t|
 end
 
 task :test do
```

Comment:
you can remove this whole task now. the `test` task is defined by default to invoke `Rake::Task[:testing]`